impl/src/rendering/templates/partials/rust_library.template

```diff
@@ -12,16 +12,20 @@ alias(
 rust_library(
     name = "{{ target_name_sanitized }}",
 {% include "templates/partials/common_attrs.template" %}
-    # buildifier: leave-alone
+{%- set deps = [] %}
+{%- if crate.build_script_target %}
+    {%- set deps = deps | concat(with=":" ~ crate_name_sanitized ~ "_build_script") %}
+{%- endif %}
+{%- for dependency in crate.default_deps.dependencies %}
+    {%- set_global deps = deps | concat(with=dependency.buildable_target) %}
+{%- endfor %}
+{%- for dependency in crate.raze_settings.additional_deps %}
+    {%- set_global deps = deps | concat(with=dependency) %}
+{%- endfor %}
+    # buildifier: leave-alone{# TODO: The sort below is different from what Buildifier does. We should write a Buildifier compliant sort function and register it so we can use it below. See https://tera.netlify.app/docs#filters for details #}
```

Comment:
This seems pretty verbose for something being included in every single build invocation. I'm imagining downstream users running a `raze update` and seeing this get generated into every single library and binary.

How do you feel about either (1) dropping this completely, or (2) referencing a github issue instead?

Reply:
I like `(2)` but I might take some time over the weekend to try and write that custom filter. I try to avoid spending too much time on how these files get rendered because it's pretty time consuming but having these deps sorted seemed important to folks in other PRs issues so I thought I'd give it a shot since I had recently learned how to group and sort a list in a template.

Reply:
Are you ok with just dropping this comment then? IIUC this PR is blocking the Bazel one, and I intend to cut a release between this PR being submitted and that one, so whatever this generates is going to end up in user BUILD files.

Reply:
This PR is not blocking https://github.com/google/cargo-raze/pull/340 but I can quickly make a new issue and update the comment

Reply:
Updated the comments and opened https://github.com/google/cargo-raze/issues/348